.github/workflows/build-devblobstore.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Dev/Test Blobstore Docker Image
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - '.github/workflows/build-devblobstore.yml'
+
+  workflow_dispatch: 
+
+env:
+  MINIO_VERSION: latest
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+    
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Docker Build & Push
+      uses: docker/build-push-action@v6
+      env:
+        DOCKER_BUILD_NO_SUMMARY: true
+      with:
+        context: .
+        file: docker/devblobstore.Dockerfile
+        platforms: linux/amd64,linux/arm64
+        push: true
+        build-args: MINIO_VERSION=${{ env.MINIO_VERSION }}
+        tags: |
+          ghcr.io/ietf-tools/datatracker-devblobstore:${{ env.MINIO_VERSION }}
```

Comment:
this will tag as :latest twice when MINIO_VERSION=latest. That will probably work, but maybe want a `minio-` prefix on that one?

Reply:
(result of out-of-band discussions): we'll leave this be unless it turns out to not work as expected.